include/ui/form.php

```diff
@@ -87,6 +87,26 @@ function bd_render_sticky_settings( $slug ) {
 <?php
 }
 
+/**
+ * Check there is have any private post is exist.
+ *
+ * @since 6.0.0
+ *
+ * @param string $post_type.
+ */
+function are_private_posts_present( $post_type='any') {
+	$args  = array(
+		'post_status' => array( 'private' ),
+		'post_type'   => $post_type,
+	);
+	$query = new WP_Query( $args );
+	if ( $query->have_posts() ) {
```

Comment:
Replace the below statements with just `return $query->have_posts()`

Reply:
@sudar fixed